api/api_ethereum.go

```diff
@@ -1453,7 +1453,46 @@ func (api *EthereumAPI) PendingTransactions() ([]*EthRPCTransaction, error) {
 // Resend accepts an existing transaction and a new gas price and limit. It will remove
 // the given transaction from the pool and reinsert it with the new gas price and limit.
 func (api *EthereumAPI) Resend(ctx context.Context, sendArgs EthTransactionArgs, gasPrice *hexutil.Big, gasLimit *hexutil.Uint64) (common.Hash, error) {
-	return common.Hash{}, errors.New("this api is not supported by Klaytn because Klaytn use fixed gasPrice policy")
+	if sendArgs.Nonce == nil {
+		return common.Hash{}, fmt.Errorf("missing transaction nonce in transaction spec")
+	}
+
+	if err := sendArgs.setDefaults(ctx, api.publicTransactionPoolAPI.b); err != nil {
+		return common.Hash{}, err
+	}
+
+	matchTx := sendArgs.toTransaction()
+	pending, err := api.publicTransactionPoolAPI.b.GetPoolTransactions()
+	if err != nil {
+		return common.Hash{}, err
+	}
+	for _, p := range pending {
+		signer := types.LatestSignerForChainID(p.ChainId())
+		wantSigHash := signer.Hash(matchTx)
+
+		if pFrom, err := types.Sender(signer, p); err == nil && pFrom == *sendArgs.From && signer.Hash(p) == wantSigHash {
```

Comment:
`sendArgs.From != nil` should be add before `pFrom == *sendArgs.From`

Reply:
I think sendArgs.From is always set from inputtransactionformatter. How do you think?

Reply:
Got it. I am content with either including or retaining it as it is.